platforms/software/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/ivyservice/resolveengine/DependencyGraphBuilderTest.groovy

```diff
@@ -1179,6 +1182,10 @@ class DependencyGraphBuilderTest extends Specification {
             components.add(node.owner.moduleVersion)
         }
 
+        @Override
+        void visitSelector(DependencyGraphSelector configurationSelector) {
```

Comment:
Since https://github.com/gradle/gradle/pull/26860, this method should not be necessary